data/dbModels/users.js

```diff
@@ -8,13 +8,26 @@ async function create(user) {
   return result[0];
 }
 
-function readBySlackId(slack_mem_id) {
-  return db('users')
-    .where({ slack_mem_id })
-    .first();
+function read(slack_mem_id = null, org_id = null) {
+  if (slack_mem_id) {
+    return db('users')
+      .where({ slack_mem_id })
+      .first();
+  }
+
```

Comment:
I'm not comfortable having this read user also handle read organization users, please can we adhere to the single responsibility principle we adopted earlier, get back the model how it was and create another model for this read users by organization, thanks

Reply:
Someone already started work on this